Makefile

```diff
@@ -81,8 +81,6 @@ clean:
 export TP_DEPLOYMENT_YAML ?=./manifests/0000_20_kube-apiserver-operator_06_deployment.yaml
 export TP_CMD_PATH ?=./cmd/cluster-kube-apiserver-operator
 
-$(call verify-golang-versions,Dockerfile.rhel7)
```

Comment:
where is this duplicated?

Reply:
we already have that done inside build-machinery-go https://github.com/openshift/build-machinery-go/blob/master/make/golang.mk#L11

Reply:
line 37 - https://github.com/openshift/cluster-kube-apiserver-operator/blob/22f2a00c6293c3a979804651e4446a57ec331ed6/Makefile#L37